experimental/FTheoryTools/src/auxiliary.jl

```diff
@@ -14,6 +14,8 @@ end
 # 2: Construct ambient space from given base
 ################################################################
 
+_ambient_space_from_base(base::ToricCoveredScheme) = _ambient_space_from_base(underlying_toric_variety(base))
+
 function _ambient_space_from_base(base::AbstractNormalToricVariety)
```

Comment:
Does the return value of this function also provide information about the embedding of the original variety? If not, can we get that information through other channels?

Reply:
Implicitly, you can read-off the embedding from comparing the `cox_ring` of `base` and `ambient_space`, but this is not (at least not yet) very explicit. We can consider a (toric) morphism for this at some later stage.